### ISSUE TITLE
Set up http://purl.unep.org/sdg/imports/

### ISSUE THREAD

**Issue by cmungall** [SDG-InterfaceOntology/sdgio#46]
We need http://purl.unep.org/sdg/imports/ to redirect to https://raw.githubusercontent.com/SDG-InterfaceOntology/sdgio/master/src/imports/ (partial redirect)

Can whoever is in charge of the purl server do this?

note I favor an alternative to the persistenturls software that will make this easier. See:
https://github.com/SDG-InterfaceOntology/sdgio/issues/2#issuecomment-158270805

**Comment by cmungall** [comment]
Can you also make the following changes

    http://purl.unep.org/sdg/sdgio.owl -->  https://raw.githubusercontent.com/SDG-InterfaceOntology/sdgio/master/sdgio.owl
    http://purl.unep.org/sdg/releases/ --PARTIAL-REDIRECT->  https://raw.githubusercontent.com/SDG-InterfaceOntology/sdgio/v

**Comment by pbuttigieg** [comment]
@cmungall:
Is there an error here?:
`http://purl.unep.org/sdg/releases/ --PARTIAL-REDIRECT->  https://raw.githubusercontent.com/SDG-InterfaceOntology/sdgio/v`

**Comment by cmungall** [comment]
No, this is intentional. On github the release tags are vYYYY-MM-DD, the 
release purls are ".../releases/YYYY-MM-DD"

ENVO is also set up this way, see:
https://github.com/EnvironmentOntology/envo/blob/master/src/envo/README-editors.md

**Comment by mark-jensen** [comment]
@cmungall @pbuttigieg Just to make clear for future reference
http://purl.unep.org/sdg/imports/XXX
should redirect to
https://raw.githubusercontent.com/SDG-InterfaceOntology/sdgio/master/src/imports/XXX

Example:
http://purl.unep.org/sdg/imports/envo_import.owl 
should find
https://raw.githubusercontent.com/SDG-InterfaceOntology/sdgio/master/src/imports/envo_import.owl

Which currently doesn't happen btw. Providing this is correct and we want the imports PURLd, I'll forward to UNEP to fix.

**Comment by cmungall** [comment]
That would work for now, but I would like to switch to the same flow as 
other ontologies, where src/ is just the source files for the working 
version, and the top level is

sdgio.owl
imports/*.owl
